src/modules/uxrce_dds_client/dds_topics.h.em

```diff
@@ -46,7 +46,7 @@ struct SendSubscription {
 struct SendTopicsSubs {
 	SendSubscription send_subscriptions[@(len(publications))] = {
 @[    for pub in publications]@
-			{ uORB::Subscription(ORB_ID(@(pub['topic_simple']))),
+			{ uORB::SubscriptionInterval(ORB_ID(@(pub['topic_simple'])), @(pub['interval_us'])),
```

Comment:
I was a bit concerned with making this as cheap as possible because ultimately I want to expose as many topics as possible. If needed we could optimize this later to use simple subscriptions.